### PR TITLE
Lonestar Pricing Overhaul (Testing) 

### PR DESCRIPTION
**Pull request by Jul-Jul**
## About The Pull Request
<summary>
This PR lowers all the markup pricing for all stations across the board aside from some unique stations, such as the Bluespace station. 

To put it simply, most station item prices' have been decreased, and stations will give more money when selling to stations. The EXPORT function (Upon pressing the Export button with the cooldown) has not been touched.

The purpose of this PR is in hopes of fixing the economy (one can wish) by making more common items more easily accessible and purchasable from Lonestar via the stations, whilst the stations should give more reasonable pricing for items. 

Side Note! Nothing within this PR is set in stone and may change and differ depending on feedback and requests from upper staff. 

Selling to a station no longer gives the station the money its paying you.
Selling to a station no longer gives favour at this time
Favour is now dobled per credit
Favour requirements have been reduced heavilly per rank to help people better get into better stations

### CODE REVIEW

**Comment by Jul-Jul** [comment]
Also yes, I do intend to Squash the Commits once I get home. Don't worry.

**Comment by Trilbyspaceclone** [comment]
After a bit more testing with the ore ship and seeing how that goes i got no reason to not merge this

File: code/modules/trade/datums/_trade_station.dm [hunk 1]
```diff
@@ -22,8 +22,8 @@
 	var/list/name_pool = list()
 
 	var/markup = WHOLESALE_GOODS
-	var/markdown = 0.8				// Default markdown is 20% - SoJ edit they get less markdown
-	var/favour_purchase_ratio = 0.25
+	var/markdown = 0.95				 // Default markdown is 20% - SoJ edit they get less markdown. Editted by Ali for testing to 5%.
+	var/favour_purchase_ratio = 0.5  // For every credit we gain half a favour point
```
Comment (Trilbyspaceclone) [inline code/modules/trade/datums/_trade_station.dm:26]:
```suggestion
	var/favour_purchase_ratio = 0.25  // 4 credits = 1 favour
```

File: code/modules/trade/datums/_trade_station.dm [hunk 2]
```diff
@@ -22,8 +22,8 @@
 	var/list/name_pool = list()
 
 	var/markup = WHOLESALE_GOODS
-	var/markdown = 0.8				// Default markdown is 20% - SoJ edit they get less markdown
-	var/favour_purchase_ratio = 0.25
+	var/markdown = 0.95				 // Default markdown is 20% - SoJ edit they get less markdown. Editted by Ali for testing to 5%.
```
Comment (Trilbyspaceclone) [inline code/modules/trade/datums/_trade_station.dm:25]:
```suggestion
	var/markdown = 0.8				 // Default markdown is 20% - SoJ edit they get less markdown.
```

**Comment by Guidesu** [comment]
Very good changes.
👍

Sheep head approved.

**Pull request merged**